lib/LANraragi/Plugin/Download/EHentai.pm

```diff
@@ -4,6 +4,7 @@ use strict;
 use warnings;
 no warnings 'uninitialized';
 
+use URI;
```

Comment:
We don't really install the `URI` module as a dependency in the cpanfile, it's brought in by our dependency on `URI::Escape`.
It seems `URI::Escape` was basically replaced/included in the full URI pod if https://metacpan.org/pod/URI is any indication, so the matching line in the cpanfile should be modified: https://github.com/Difegue/LANraragi/blob/8e2114bc77e3adff0d4cf1cc4adf30b8198ac1b5/tools/cpanfile#L19 
`requires 'URI', 5.09;`

Reply:
honestly thought it was part of the stdlib and thought nothing of it